src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no,  initial-scale=1.0, maximum-scale=1.0, minimum-scale=1. 0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+  </head>
+  <body>
+    <div class="card card--main" data-qa="card">
+      <div class="card__pic_container">
+        <img
+          class="card__pic" 
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K   
+          Monoblock"
+        >
+      </div>
+      <div class="card__about card__about--font">
+        APPLE A1419 iMac 27" Retina 5K   
+        Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__prod_id card__prod_id--font">
+        Product code: 195434
+      </div>
```

Comment:
It is okay to add new lines between big adjacent blocks of code. 
Just don't add them everywhere. Your code should be clean and easily readable for other developers
```suggestion
      <div class="card__pic_container">
        <img
          class="card__pic" 
          src="./images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K   
          Monoblock"
        >
      </div>
      
      <div class="card__about card__about--font">
        APPLE A1419 iMac 27" Retina 5K   
        Monoblock (MNED2UA/A)
      </div>
      
      <div class="card__prod_id card__prod_id--font">
        Product code: 195434
      </div>
```